src/app/clusters/door-lock-server/door-lock-server.cpp

```diff
@@ -12,8 +12,7 @@
  *    distributed under the License is distributed on an "AS IS" BASIS,
  *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  *    See the License for the specific language governing permissions and
- *    limitations under the License.
- */
+ *    limitations under the License. */
```

Comment:
Why this change?

Reply:
Not sure. I was thinking it might have been clang-format but it doesn't seem touch that line either way. I'll change it back.